src/lib.rs

```diff
@@ -725,6 +728,19 @@ pub struct Record<'a> {
     module_path: Option<&'a str>,
     file: Option<&'a str>,
     line: Option<u32>,
+    #[cfg(feature = "kv_unstable")]
+    key_values: KeyValues<'a>,
+}
+
+#[cfg(feature = "kv_unstable")]
+#[derive(Clone)]
+struct KeyValues<'a>(&'a key_values::Source);
```

Comment:
Please add a comment that the reason this wrapper exists is that putting `key_values: &Source` inside Record would prevent using derive(Debug).